2020-01-Winter/0_arrays_sorting/README.md

```diff
@@ -45,19 +45,37 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Search in Rotated Sorted Array :bug:
 
-Source: TODO :bug:
+Source: <a href="https://leetcode.com/problems/search-in-rotated-sorted-array/">LeetCode</a> :bug:
```

Comment:
Consider using native Markdown instead of HTML.

```suggestion
Source: [LeetCode](https://leetcode.com/problems/search-in-rotated-sorted-array/)
```